Tools/CISupport/ews-build/steps.py

```diff
@@ -799,6 +799,7 @@ def run(self):
         rebase_target_hash = self.getProperty('ews_revision') or self.getProperty('got_revision')
 
         commands = [
+            ['git', 'config', 'credential.helper', '!echo_credentials() { sleep 1; echo "username=${GIT_USER}"; echo "password=${GIT_PASSWORD}"; }; echo_credentials'],
```

Comment:
where is GIT_PASSWORD coming from?

Reply:
Assuming this is coming from environment variable on the server, wouldn't it be simpler to just configure this on the bot manually, instead of running this in every build?

Reply:
This is very similar to what we're doing with `merge.changelog.driver`.

Basically, this command lets us pass credentials to future `git` commands by setting those credentials as environment variables `GIT_USER` and `GIT_PASSWORD`. This is an important ability because it means that workers _do not_ have credentials on disk, and only get access to credentials when they're running commands that need those credentials. #228 is the change which leverages this, you can see that because we pass `GIT_USER` and `GIT_PASSWORD` as environment variables. In the relatively near future, we'll need to do that when we checkout pull requests as well, because those pull requests may be coming from secure remotes.